### ISSUE TITLE
[Suggestion] Updater

### ISSUE THREAD

**Issue by KnuffelBeestje** [MillenniumEarl/YAM#53]
**Is your feature request related to a problem? Please describe.**
It can be annoying sometimes when you're still on an oudated version.

**Describe the solution you'd like**
Create an update button in the settings page. So you still have a choice if you want to download/install the update or not.

**Describe alternatives you've considered**
An autoupdater maybe, but not that many people like autoupdaters.

**Comment by MillenniumEarl** [comment]
Hi @KnuffelBeestje,

The idea of an autoupdater (or in any case of an update notification) had already occurred to me some time ago. At first I used the already included functionality of `electron-builder` with the help of TravisCI. The only problem was that in those days Travis [had started to limit the number of operations](https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing) that could be performed monthly and since for each commit there are three builds (Mac/Windows/Linux) just for testing (in one day) I had already burned 3000 of the 10,000 guaranteed as a test.

Then I tried another free service ([UpdateRocks](https://www.update.rocks)) and created the [updater.js](https://github.com/MillenniumEarl/YAM/blob/main/app/src/scripts/updater.js) file. However, this method requires you to change the type of installer from NSIS to Squirrel, the latter being considered "[deprecated](https://www.electron.build/configuration/squirrel-windows)" by `electron-builder`. I'll try to do some testing and see if Squirrel can be an alternative, otherwise I'll make do and write a script for it.
